src/main/kotlin/me/owdding/skyblockpv/api/data/SkyBlockProfile.kt

```diff
@@ -122,7 +121,7 @@ data class SkyBlockProfile(
                 rift = playerStats?.getAsJsonObject("rift")?.let { stats -> RiftData.fromJson(member.getAsJsonObject("rift"), stats) },
                 crimsonIsleData = CrimsonIsleData.fromJson(member.getAsJsonObject("nether_island_player_data")),
                 // todo: asStringList or smth
-                minions = playerData?.getAsJsonArray("crafted_generators").asList { it.asString("") }.filterNot { it.isEmpty() }
+                minions = playerData?.getAsJsonArray("crafted_generators").asStringList().filterNot { it.isEmpty() }
```

Comment:
I think we could add the filter not empty to the `asStringList` and also make the receiver nullable so that both this and line 104 are shorter